test/tests/functional/pbs_cray_vnode_pool.py

```diff
@@ -49,6 +49,11 @@ class TestVnodePool(TestFunctional):
 
     def setUp(self):
         TestFunctional.setUp(self)
+        try:
```

Comment:
@hirenvadalia, I reconsidered this and the tests should be skipped if the required test environment is not there.